### DISCUSSION TITLE
How to validate contenteditable field

### DISCUSSION THREAD

**Discussion by cloud2torial** [react-hook-form/react-hook-form#2507] [Q&A] [answered]
`<h1 className={Styles.ctPageHeaderTitle} contentEditable="true"  ref={register}>`

**Comment by bluebill1049** [comment, marked as answer]
you have to do custom `register` at `useEffect` and update with `setValue`

**Reply by cloud2torial** [reply]
But how can i show min max error messages?

**Reply by cloud2torial** [reply]
i tried this
 register({ name: 'screen_title' }, { required: true })

**Reply by bluebill1049** [reply]
```
register(`screen_title', { required: true })

setValue('screen_title', 'your value here', { shouldValidate: true })
```

everything is in the doc, have a good read there, it will help :)

**Reply by cloud2torial** [reply]
ok thanks
